File: QuickApp.Web/src/shared/models/session/userSessionInfoDto.ts
import { PermissionDto } from "@shared/models/session/permissionDto";
export class UserSessionInfoDto {
    Username: string;
    FirstName: string;
    LastName: string;
    EmailAddress: string;
    Location: string;
    GrantedPermissions: PermissionDto[];
    constructor() { }
}

File: QuickApp.Core/Dto/User/UserSessionInfoDto.cs
using System.Collections.Generic;
using QuickApp.Core.Authorization;

namespace QuickApp.Core.Dto.User
{
    public class UserSessionInfoDto
    {
        public string Username { get; set; }

        public string FirstName { get; set; }

        public string LastName { get; set; }

        public string EmailAddress { get; set; }

        public string Location { get; set; }

        public List<PermissionDto> GrantedPermissions { get; set; }
    }
}
